Meshtastic/Views/Nodes/Helpers/Map/MapContent/MeshMapContent.swift

```diff
@@ -24,6 +29,11 @@ struct MeshMapContent: MapContent {
 	@AppStorage("enableMapWaypoints") private var showWaypoints = true
 	@Binding var selectedWaypoint: WaypointEntity?
 
+	// Burning Man GeoJSON overlays
+	@AppStorage("burningManShowStreets") private var showStreets = false
```

Comment:
Todo: consider making these a single config and then show/hide all "Event" assets.

(ideally, default enabled if this can be conditionally hidden unless someone zooms in on that region of the map containing the event)